internal/migrations/20220325125504_multiple_networks_cleanup_test.go

```diff
@@ -0,0 +1,81 @@
+package migrations
+
+import (
+	"github.com/go-openapi/strfmt"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/models"
+	"gorm.io/gorm"
+)
+
+var _ = Describe("multipleNetworksCleanup", func() {
+	var (
+		db            *gorm.DB
+		dbName        string
+		clusterID     strfmt.UUID
+		cluster       *common.Cluster
+		clusterFromDb *common.Cluster
+	)
+
+	BeforeEach(func() {
+		db, dbName = common.PrepareTestDB()
+		clusterID = strfmt.UUID(uuid.New().String())
+		cluster = &common.Cluster{
+			Cluster: models.Cluster{
+				ID:              &clusterID,
+				ClusterNetworks: common.TestDualStackNetworking.ClusterNetworks,
+				ServiceNetworks: common.TestDualStackNetworking.ServiceNetworks,
+				MachineNetworks: common.TestDualStackNetworking.MachineNetworks,
```

Comment:
Yes, if in the test setup we create cluster with 0 networks, then the fields that we are emptying as this migration will be empty from the beginning, so we will never test the code that empties them.

The point of this test is so that we create a cluster with _some_ networks in it, then run the code that removes those and then we check in the DB that they have been removed

Reply:
E.g. if we removed those networks here, then test
```
db.Raw("SELECT cluster_network_cidr FROM clusters WHERE id = ?", clusterID).Scan(&value)
Expect(value[0]).To(Equal("1.3.0.0/16"))
```
needs to be removed, but then

```
db.Raw("SELECT cluster_network_cidr FROM clusters WHERE id = ?", clusterID).Scan(&valueStr)
Expect(valueStr[0]).To(Equal(""))
```
does not really test our migration

Reply:
I mean specifically these three values.
We're setting them here and then just testing that they're still set after migrating.

These are not the fields we're emptying, right? Those are all set up in the raw sql lines I think.